### PR TITLE
fixing readme for properties

### PR DESCRIPTION
**Pull request by jeffgrunewald**
I don't know how you'd do this accidentally, so I assume it was done on purpose to irritate my OCD.

### CODE REVIEW

File: apps/properties/README.md [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
 
 ## Usage
 
 ```elixir
   defmodule Example do
-    use ExUnit.Case
-    import AssertAsync
+    use Properties, otp_app: :example_service
+
+    @app_name get_config_value(:app_name, required: true)
```
Comment (bbalser) [inline apps/properties/README.md]: Should we be clear about not using this format unless absolutely necessary?

Comment (jeffgrunewald) [reply apps/properties/README.md]: which format specifically? using get_config_value and binding it to a module attribute?

Comment (bbalser) [reply apps/properties/README.md]: yeah, the goal is to make it painless to access the config at runtime and get_config_value should only be used when the config has to be done a compile time.

Comment (jeffgrunewald) [reply apps/properties/README.md]: sure thing.

**Pull request merged**